DataManagementSystem/Client/FTSFile.py

```diff
@@ -76,7 +76,7 @@ def tableDesc():
                "FileID": "INTEGER NOT NULL",
                "OperationID": "INTEGER NOT NULL",
                "RequestID": "INTEGER NOT NULL",
-               "LFN": "VARCHAR(255) NOT NULL",
+               "LFN": "VARCHAR(955) NOT NULL",
```

Comment:
LFN by design is smaller length than any SURL.
Any LFN with 1024 char will not fit in SURL.
955 is not an ideal choice either. Suggestions are welcome.